platform/viewer/cypress/integration/common/OHIFUserPreferences.spec.js

```diff
@@ -0,0 +1,571 @@
+describe('OHIF User Preferences', () => {
+  context('Study List Page', function() {
+    before(() => {
+      cy.visit('/');
+    });
+
+    beforeEach(() => {
+      // Open User Preferences modal
+      cy.openPreferences().then(() => {
+        cy.initPreferencesModalAliases();
+      });
+    });
+
+    it('opens User Preferences modal and verify the displayed information', function() {
+      cy.get('@preferencesModal').should('be.visible');
+      cy.get('.HotKeysPreferences').should('be.visible');
+      cy.get('@hotkeysLink').should('have.class', 'active');
+      cy.get('@generalLink').should('be.visible');
+      cy.get('@restoreBtn')
+        .scrollIntoView()
+        .should('be.visible');
+      cy.get('@cancelBtn').should('be.visible');
+      cy.get('@saveBtn').should('be.visible');
+
+      // Visual comparison
+      cy.screenshot(
+        'User Preferences Modal - Hotkeys tab initial state in Study Viewer page'
+      );
+      cy.percyCanvasSnapshot(
+        'User Preferences Modal - Hotkeys tab initial state in Study Viewer page'
+      );
+      cy.get('.close').click();
+    });
+
+    it('checks translation by selecting Spanish language', function() {
+      cy.get('@generalLink')
```

Comment:
Ah. Okay. Instead of `@generalLink` can we do something more like `@userPreferencesGeneralTab` or something similarly descriptive?